### PR TITLE
Checkbox test suite

### PR DESCRIPTION
**Pull request by juwara0**

### CODE REVIEW

File: tests/unit/components/frost-checkbox-test.js [hunk 1]
```diff
@@ -17,53 +27,69 @@ describeComponent(
...
-          run(() => {
-            component.set('checked', test.in)
-          })
-          expect(component.get('isChecked')).to.equal(test.out)
-        })
-      })
+    it('sets the inputId', function () {
+      const testId = 99
+
+      run(() => component.set('id', testId))
```
Comment (sglanzer-fire) [inline tests/unit/components/frost-checkbox-test.js]: With the presence of hooks, I'm not sure we should even really bother with ids anymore - just treat them as controlled by Ember and keep our hands off

File: tests/unit/components/frost-checkbox-test.js [hunk 2]
```diff
@@ -79,5 +105,91 @@ describeComponent(
         }).not.to.throw(Error)
       })
     })
+
+    describe('keyPress()', function () {
```
Comment (sglanzer-fire) [inline tests/unit/components/frost-checkbox-test.js:100]: It's specifically spacebar, correct?

Comment (juwara0) [reply tests/unit/components/frost-checkbox-test.js:100]: Yes

File: tests/unit/components/frost-checkbox-test.js (hunk 1 above)
Comment (juwara0) [reply tests/unit/components/frost-checkbox-test.js]: Done

**Pull request merged**